src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,43 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+
+    private static final int MINUS = -1;
+    private static final int FULL = 12;
+    private int[] numbers = new int[FULL];
+
+
+    private int total = MINUS;

```

Comment:
Extracting this constant is done incorrectly. Should show during next classes ;)